### ISSUE TITLE
glibc for ubuntu2004 ?

### ISSUE THREAD

**Issue by colindaven** [stjude-rust-labs/fq#38]
Hi,

this package looks nice for fastq validation, thanks.

I'm getting errors with the released binary version since my Glibc version on ubuntu 20.04 is only 2.31. Are you open to extending compatibility to this slightly older glibc version ? I saw the README mentioned only Ubuntu 14.04+ is needed so was puzzled (was this for the self compiled version?). Else I'll have to install rust and cargo in the container and compile there, but this is less elegant and more time consuming than a wget. 

Thanks,
Colin

```
Command error:
  fq: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by fq)
  fq: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by fq)
  fq: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by fq)
```

**Comment by colindaven** [comment]
Ah, never mind, I missed that it was in bioconda already and this covers my use case.

**Comment by zaeleus** [comment]
Thanks for reporting. The README became outdated as workflow runners changed over time. fq 0.12.0 is now released and built on Ubuntu 20.04, so it now requires glibc 2.31 instead of 2.35. As you mentioned, the conda-provided package is also a viable alternative.
